test/powershell/engine/Module/IsolatedModule.Tests.ps1

```diff
@@ -27,10 +27,12 @@ Describe "Isolated module scenario - load the whole module in custom ALC" -Tag '
         $context1.Name | Should -BeExactly "MyCustomALC"
         $context1 | Should -Be $context2
 
-        ## Test-NestedCommand depends on NewtonSoft.Json 10.0.0.0 while PowerShell depends on 13.0.0.0 or higher.
+        ## Test-NestedCommand depends on NewtonSoft.Json 13.0.1.0 while PowerShell depends on 13.0.3.0 or higher.
         ## The exact version of NewtonSoft.Json should be loaded to the custom ALC.
         $foo = [Test.Isolated.Nested.Foo]::new("Hello", "World")
-        Test-NestedCommand -Param $foo | Should -BeExactly "Hello-World-Newtonsoft.Json, Version=10.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed"
+        $location = Test-NestedCommand -Param $foo
+        $pathSeparator = [System.IO.Path]::DirectorySeparatorChar
+        $location | Should -BeLikeExactly "*${pathSeparator}test${pathSeparator}tools${pathSeparator}Modules${pathSeparator}IsolatedModule${pathSeparator}Dependencies${pathSeparator}Newtonsoft.Json.dll"
```

Comment:
Have we a tracking issue for this?
(Isn't it time to get rid of it altogether? It seems to be causing more trouble than it's worth.)

Reply:
The change here is that Newtonsoft.Json has a security issue that is called out during CI.  So looking at the CVE, you need version 13.0.0.1 or higher so that breaks the original test.  I think this change is fine to show multiple versions of Newtonsoft loaded at once in separate ALCs.